modules/loggers/mod_log_config.c

```diff
@@ -263,7 +277,9 @@ typedef struct {
     const char *fname;
     const char *format_string;
     apr_array_header_t *format;
-    void *log_writer;
+    void *log_writer; /* log_writer_data */
```

Comment:
Maybe do the renaming then? `log_writer` the function vs `log_writer` the data is quite confusing..